src/render.js

```diff
@@ -1,35 +1,49 @@
+// Sections are groups of changelog entries by category
+module.exports = function renderSections(sections, format = 'md') {
+    if (format === 'md') {
+        return renderSectionsMd(sections);
+    } else if (format === 'json') {
+        return renderSectionsJson(sections);
+    } else {
+        throw new Error(`Unknown format requested: ${format}`);
+    }
+}
+
+function renderSectionsMd(sections) {
+    let output = '';
+
+    for (const section of Object.values(sections)) {
+        if  (section.entries.length > 0) {
+            output += `## ${section.title}\n`;
+            for (const entry of section.entries) {
+                output += renderEntryMd(entry)
+                output += '\n';
+            }
+            output += '\n\n';
+        }
+    }
+
+    return output;
+}
+
+function renderSectionsJson(sections) {
+    return JSON.stringify(sections);
+}
 
+function renderEntryMd(entry) {
+    const description = entry.body;
+    const pr = entry.pullRequest;
 
-exports.commitSummary = function commitSummary(commit) {
-    const {issues, pr} = commit;
-    const title = pr ? formatIssue(pr) : `${commit.title} ([${commit.sha.slice(0, 7)}](${commit.url}))`;
+    const prRef = ` ([#${pr.number}](${pr.html_url}))`;
 
     let hattip = '';
-    if (pr && pr.user) {
-        hattip = ` (h/t [${pr.user.login}](${pr.user.url}))`;
+    if (pr.head.user.login !== pr.base.user.login) {
+        hattip += ` (h/t [${pr.user.login}](${pr.user.url}))`;
```

Comment:
This links to `api.github.com` but should link to the HTML profile on `github.com`